Formula/libobjc2.rb

```diff
@@ -0,0 +1,57 @@
+class Libobjc2 < Formula
+  desc "Objective-C runtime library intended for use with Clang"
+  homepage "https://github.com/gnustep/libobjc2"
+  url "https://github.com/gnustep/libobjc2/archive/refs/tags/v2.1.tar.gz"
+  sha256 "78fc3711db14bf863040ae98f7bdca08f41623ebeaf7efaea7dd49a38b5f054c"
+  license "MIT"
+
+  depends_on "cmake" => :build
+  depends_on "pkg-config" => :test
+  # Clang explicitly forbids building Mach-O binaries of libobjc2.
+  # https://reviews.llvm.org/D46052
```

Comment:
Where in the link does it say/imply this?

Reply:
I mostly got it from the code added in the commit:
```
    if ((runtime.getKind() == ObjCRuntime::GNUstep) &&
        (runtime.getVersion() >= VersionTuple(2, 0)))
      if (!getToolChain().getTriple().isOSBinFormatELF()) {
        getToolChain().getDriver().Diag(
            diag::err_drv_gnustep_objc_runtime_incompatible_binary)
          << runtime.getVersion().getMajor();
      }
```
and 
```
def err_drv_gnustep_objc_runtime_incompatible_binary : Error<
  "GNUstep Objective-C runtime version %0 incompatible with target binary format">;
```
It still produces this error if I try to build it now, so I guess the limitation still applies as LLVM 14.